packages/ng/readMore/read-more.component.ts

```diff
@@ -0,0 +1,75 @@
+import { booleanAttribute, Component, ElementRef, HostBinding, input, OnInit, signal, viewChild, ViewEncapsulation } from '@angular/core';
+
+@Component({
+	selector: 'lu-read-more',
+	standalone: true,
+	templateUrl: './read-more.component.html',
+	styleUrls: ['./read-more.component.scss'],
+	encapsulation: ViewEncapsulation.None,
+
+	host: {
+		class: 'readMore',
+	},
+})
+export class ReadMoreComponent implements OnInit {
+	lineClamp = input<number>(5);
+	openOnly = input(false, { transform: booleanAttribute });
+	surface = input<null | 'sunken' | 'raised' | string>(null);
+
+	label = signal<string>('Lire plus');
```

Comment:
Is it actually translated somewhere?

Reply:
No, but I actually had to do it! Thank you.